lib/dialects/postgres/connection-manager.js

```diff
@@ -109,18 +110,21 @@ class ConnectionManager extends AbstractConnectionManager {
           return;
         }
         responded = true;
+        debug(`connection acquire`);
```

Comment:
For `postgres` even during transactions we don't have `uuid` in connection object. But query have used `this.client` object whose access we don't have in `connection-manager.js`